api/ccapi/include/layer.h

```diff
@@ -39,6 +40,7 @@ enum class LayerType {
   LAYER_ADDITION,                       /** Addition Layer type */
   LAYER_CONCAT,                         /** Concat Layer type */
   LAYER_LOSS,                           /** Loss Layer type */
+  LAYER_BACKBONE_NNSTREAMER,            /** Backbone using NNStreamer */
```

Comment:
Are you going to add nntrainer backbone here as well?

Reply:
nntrainer backbone does not need a type.
it directly modifies the graph (layer vector for now).

Reply:
> nntrainer backbone does not need a type.
> it directly modifies the graph (layer vector for now).

That means if user add sub .ini file in backbone, then it parse and initialize?

Reply:
yes, the backbone ini file is parsed and a backbone graph is made.
Then the backbone graphs is merged into the original graph.